packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -65,8 +64,18 @@ const entryPointExists = (segments) => {
 }
 
 const findInProjectThenSDK = (pkg) => {
-    const projectPath = resolve(projectDir, 'node_modules', pkg)
-    return fs.existsSync(projectPath) ? projectPath : resolve(sdkDir, 'node_modules', pkg)
+    const candidates = [
+        resolve(projectDir, 'node_modules', pkg),
+        resolve(__dirname, '..', '..', 'node_modules', pkg),
+        resolve(__dirname, '..', '..', '..', 'node_modules', pkg)
```

Comment:
Good finding @raiyaj! 
It looks like we don't need the last one since we saw that generates the wrong path with the duplicated `node-modules` folder.

Reply:
Thanks Adam! I was thinking of removing it initially, but I added some logging [here](https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/4240394834/jobs/7369372170), and saw that when a project is generated in CI, `__dirname` is `/home/runner/work/pwa-kit/pwa-kit/packages/pwa-kit-dev/dist/configs/webpack`, whereas locally it's `/Users/rjessa/generated-test-project/node_modules/pwa-kit-dev/configs/webpack`, so we need to try both options.

I copied the `candidates` idea from [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/pwa-kit-dev/src/utils/script-utils.ts#L55), where we do something similar.

Reply:
The method's name `findInProjectThenSDK` implies two places but we now look at three places. 
It can be confusing for anyone else looking at the logic or facing a similar problem in the future.

@raiyaj Do you mind leaving a comment in the code explaining why we need to do this to support NPM 7 and Node 16?